packages/webapp/src/components/Animals/RemoveAnimalsModal/styles.module.scss

```diff
@@ -0,0 +1,123 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+form > * {
+  margin-top: 24px;
+
+  &:first-child {
+    margin-top: 0;
+  }
+}
+
+fieldset {
+  all: unset;
+  width: 100%;
+}
+
+.mobileOption {
+  display: flex;
+  cursor: pointer;
+  margin-top: 4px;
+  height: 40px;
+
+  input {
+    appearance: none;
+  }
+
+  &Label {
+    height: 100%;
+    width: 100%;
+    border-radius: 8px;
+    padding: 8px 16px;
+  }
+
+  &LabelGreen {
+    border: 1px solid var(--teal600);
+    color: var(--teal600);
+  }
+
+  &LabelRed {
+    border: 1px solid #ffa9a6;
+    color: var(--red700);
+  }
+}
+
+input:checked {
+  + .mobileOptionLabelGreen {
+    font-weight: 700;
+    background-color: var(--green100);
+  }
+  + .mobileOptionLabelRed {
+    border-color: var(--red700);
+    background-color: var(--red200);
+    color: #520f0d;
+    font-weight: 700;
+  }
+}
+
```

Comment:
Minor but I think on Figma the ReactSelect has drop-shadow when selected/open.

It looks like the options also need some left-padding.

Reply:
I added the padding but wasn't sure if the background color on hover should extend past the padding, so just left it as default for now.